Content.Server/Chemistry/Metabolism/HealthChangeMetabolism.cs

```diff
@@ -0,0 +1,48 @@
+using Content.Server.GameObjects.Components.Nutrition;
+using Content.Shared.Chemistry;
+using Content.Shared.Damage;
+using Content.Shared.GameObjects.Components.Damage;
+using Content.Shared.Interfaces.Chemistry;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Interfaces.Serialization;
+using Robust.Shared.Serialization;
+
+namespace Content.Server.Chemistry.Metabolism
+{
+    /// <summary>
+    /// Default metabolism for medicine reagents. Attempts to find a DamegableComponent on the target,
+    /// and to update it's damage values.
+    /// </summary>
+    class HealthChangeMetabolism : IMetabolizable
+    {
+        //Rate of metabolism in units / second
+        private ReagentUnit _metabolismRate;
+        public ReagentUnit MetabolismRate => _metabolismRate;
```

Comment:
Are these public properties used?

Reply:
I'm not sure since I more or less copied another metabolism type to do this and just changed what I needed to. For that reason I'm not sure.